packages/ui/src/ui-component/extended/SpeechToText.jsx

```diff
@@ -16,17 +16,27 @@ import { StyledButton } from '@/ui-component/button/StyledButton'
 import { Dropdown } from '@/ui-component/dropdown/Dropdown'
 import openAISVG from '@/assets/images/openai.svg'
 import assemblyAIPng from '@/assets/images/assemblyai.png'
+import localAISVG from '@/assets/images/robot.png'
 
 // store
 import useNotifier from '@/utils/useNotifier'
 
 // API
 import chatflowsApi from '@/api/chatflows'
 
+// TODO: Import this?
+//import {SpeechToTextType} from 'flowise-components'
```

Comment:
in ui we dont have `flowise-components` dep installed, so we can't import it. for now, yeah we probably have to duplicate it

Reply:
Though a better approach is to have the nodes in `packages/components/nodes` and UI will call the server API to get the nodes specifically for STT.

That is a larger refractor thats probably outside of the scope for this PR